app/services/__mocks__/request-promise.js

```diff
@@ -0,0 +1,11 @@
+const logger = require('../../logger');
```

Comment:
the mock is perfect, but the mocks would be in the tests folder, not in the services